yarn.lock

```diff
@@ -2565,7 +2565,7 @@
   dependencies:
     fs-extra "^8.1.0"
     ssri "^6.0.1"
-    tar "^4.4.8"
+    tar "^6.1.0"
```

Comment:
These entries will be reverted by Yarn.

Reply:
the reason tar "^4.4.8" is left there is we are using  @evocateur/pacote which is using tar version 4.4.8, until they change their tar version, Yarn will always keep tar 4.4.8.